libqtile/backend/wayland/window.py

```diff
@@ -952,7 +1021,7 @@ def place(
         respect_hints: bool = False,
     ) -> None:
         if above:
-            self.bring_to_front()
+            self.move_up()
```

Comment:
@elParaguayo is this correct?

Reply:
I am confused what needs to happen when above is set in `place`

Reply:
I think `move_up` is ok here - but let's see what happens in the tests ;)